playbooks/roles/caasp4/tasks/setup_nodes.yml

```diff
@@ -14,6 +14,11 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 #
+- name: Get deployment mechanism value
+  set_fact:
+    deployment_mechanism: "{{ lookup('env', 'DEPLOYMENT_MECHANISM') }}"
```

Comment:
Could we set the deployment mechanism variable in defaults/man.yml ? That way, defaults/main.yml documents which variables a user of the role can use to influence the bahaviour of the role.

Reply:
Hm, good point. I'll change that.